src/CommunityToolkit.Aspire.Hosting.ActiveMQ/ActiveMQServerResource.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+
+namespace Aspire.Hosting.ApplicationModel;
+
+/// <summary>
+/// A resource that represents a ActiveMQ resource.
+/// </summary>
+/// <param name="name">The name of the resource.</param>
+/// <param name="userName">A parameter that contains the ActiveMQ server username, or <see langword="null"/> to use a default value.</param>
+/// <param name="password">A parameter that contains the ActiveMQ server password.</param>
+/// <param name="scheme">Scheme used in the connectionString (e.g. tcp or activemq, see MassTransit)</param>
+public class ActiveMQServerResource(string name, ParameterResource? userName, ParameterResource password,
+    string scheme) : ContainerResource(name), IResourceWithConnectionString, IResourceWithEnvironment
```

Comment:
```suggestion
public class ActiveMQServerResource(string name, ParameterResource? userName, ParameterResource password, string scheme) : ContainerResource(name), IResourceWithConnectionString, IResourceWithEnvironment
```

Let's be careful on the arbitrary line breaks